src/platforms/esp32/main/network_driver.c

```diff
@@ -236,3 +234,29 @@ static esp_err_t wifi_event_handler(void *ctx, system_event_t *event)
 
     return ESP_OK;
 }
+
+static char* get_c_str_from_list_or_binary(Context *ctx, term t)
```

Comment:
this function looks like `interop_term_to_string`. Why do we need also this?

Reply:
I guess we don't -- I hadn't seen that function.  Fixed.